manim/mobject/value_tracker.py

```diff
@@ -26,6 +27,17 @@ def set_value(self, value):
     def increment_value(self, d_value):
         self.set_value(self.get_value() + d_value)
 
+    def interpolate(self, mobject1, mobject2,
+                    alpha, path_func=straight_path):
+        """
+        Turns self into an interpolation between mobject1
+        and mobject2.
+        """
```

Comment:
I was going to comment about documentation guidelines, but I guess we can just copy the `interpolate` docs from when we finish them from Mobject into this, so should be ok